.circleci/config.yml

```diff
@@ -412,6 +412,76 @@ jobs:
       - run_tox_scenario:
           pattern: '^py..-opentracer'
 
+  profile-windows-27:
```

Comment:
Can't these be made into parametrised commands, like `run_test` etc...?

Reply:
I'm not really sure if we can nor how TBH.